### PR TITLE
feat(compare): add initial tool layout

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**
image
image

### CODE REVIEW

File: src/pages/comparison/comparison.tsx [hunk 1]
```diff
@@ -1,5 +1,268 @@
...
+
+interface layoutProps {
+  layout: string;
+}
+
+const INITIAL_VIEW_STATE = {
+  longitude: -120,
+  latitude: 34,
+  height: 600,
+  width: 800,
```
Comment (belom88) [inline src/pages/comparison/comparison.tsx:27]: `height` and `width` don't seem a part of viewState https://deck.gl/docs/api-reference/core/map-view#view-state

File: src/pages/comparison/comparison.tsx [hunk 2]
```diff
@@ -1,5 +1,268 @@
...
+          id="first-deck-container"
+          layers={layers1}
+          viewState={viewState}
+          views={[VIEW]}
+          onViewStateChange={onViewStateChange}
+          controller={{
+            type: MapController,
+            maxPitch: 60,
+            inertia: true,
+            scrollZoom: { speed: 0.01, smooth: true },
```
Comment (belom88) [inline src/pages/comparison/comparison.tsx:235]: These properties are static. Create a const for them to reuse on both sides.

File: src/pages/comparison/comparison.tsx [hunk 3]
```diff
@@ -1,5 +1,268 @@
...
+      id: `tile-layer`,
+      data: "https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/SanFrancisco_3DObjects_1_7/SceneServer/layers/0",
+      loader: I3SLoader,
+      onTilesetLoad: onTilesetLoad,
+      loadOptions,
+    }),
+  ];
+  const layers2 = [
+    new Tile3DLayer({
+      id: `tile-layer`,
```
Comment (belom88) [inline src/pages/comparison/comparison.tsx:215]: make id of `layers1` and `layers2` different.

File: src/pages/comparison/comparison.tsx [hunk 4]
```diff
@@ -1,5 +1,268 @@
...
+  maxPitch: 90,
+  bearing: 0,
+  minZoom: 2,
+  maxZoom: 30,
+  zoom: 14.5,
+  transitionDuration: 0,
+  transitionInterpolator: null,
+};
+
+const TRANSITION_DURAITON = 4000;
```
Comment (belom88) [inline src/pages/comparison/comparison.tsx:38]: Make it global for all pages

File: src/pages/comparison/comparison.tsx [hunk 5]
```diff
@@ -1,5 +1,268 @@
...
+    mobile: "100%",
+  })};
+
+  height: ${getCurrentLayoutProperty({
+    default: "100%",
+    tablet: "8px",
+    mobile: "8px",
+  })};
+
+  background-color: #232430;
```
Comment (belom88) [inline src/pages/comparison/comparison.tsx:80]: Shall we create a file for css constants?

**Review by belom88 — CHANGES_REQUESTED** [review]

File: src/pages/comparison/comparison.tsx (hunk 1 above)
Comment (dryabinin-actionengine) [reply src/pages/comparison/comparison.tsx]: removed them

File: src/pages/comparison/comparison.tsx (hunk 2 above)
Comment (dryabinin-actionengine) [reply src/pages/comparison/comparison.tsx]: done

File: src/pages/comparison/comparison.tsx (hunk 3 above)
Comment (dryabinin-actionengine) [reply src/pages/comparison/comparison.tsx]: done

File: src/pages/comparison/comparison.tsx (hunk 5 above)
Comment (dryabinin-actionengine) [reply src/pages/comparison/comparison.tsx]: done

**Review by belom88 — CHANGES_REQUESTED** [review]
Failing test

**Review by belom88 — APPROVED** [review]

**Pull request merged**